frontend/simple/views/components/Voting.vue

```diff
@@ -0,0 +1,292 @@
+        <h5 class="has-text-weight-bold is-uppercase gi-voting-info-title">{{proposal.title}}</h5>
+        <p v-html="proposal.text"></p>
+        <p class="is-size-7 has-text-grey" v-if="proposal.textDetails" v-html="proposal.textDetails"></p>
+      </div>
+    </div>
+
+    <div class="gi-voting-ctas">
+      <div class="buttons">
+        <template v-if="!proposal.ownProposal">
+          <button class="button gi-btn-against"
+            :class="{
+              'is-outlined': !hasVotedAgainst,
+              'is-danger': !hasVotedFor
+            }"
+            @click="handleVoteAgainst"
+          >
+            {{proposal.ctas.against}}
+          </button>
+
+          <button class="button gi-btn-for"
+            :class="{
+              'is-outlined': !hasVotedFor,
+              'is-success': !hasVotedAgainst
+            }"
+            @click="handleVoteFor"
+          >
+            {{proposal.ctas.for}}
+          </button>
+        </template>
+
+        <button-countdown
+          :onStateChange="handleCloseProposalStateChange"
+          v-if="proposal.ownProposal && !isProposalClosed"
+        >
+          {{closeProposalBtnText}}
+        </button-countdown>
+      </div>
+
+      <p class="has-text-grey has-text-right"
+        :class="{ 'gi-feedback has-text-weight-bold': isProposalClosed }"
+      >
+        {{helperText}}
+      </p>
+    </div>
+  </div>
+</template>
+<style lang="scss" scoped>
+@import "../../assets/sass/theme/index";
+
+%avatarSize {
+  width: 4.5rem;
+  height: 4.5rem;
+}
+
+.gi-voting {
+  flex-wrap: wrap;
+  justify-content: flex-end;
+  margin: $gi-spacer-xl 0;
+
+  &-body {
+    margin-bottom: $gi-spacer;
+    flex-grow: 1;
+  }
+
+  &-sign {
+    flex-shrink: 0;
+
+    &-avatar,
+    &-value {
+      @extend %avatarSize;
+      border-radius: 50%;
+      line-height: 4.5rem;
+      white-space: nowrap;
+
+      &.gi-is-mincome {
+        line-height: 2;
+      }
+
+      &.gi-is-rule {
+        border: 1px solid $grey-lighter;
+      }
+    }
+
+    &-svg {
+      @extend %avatarSize;
+      position: absolute;
+
+      &-circle {
+        stroke: $success;
+        stroke-width: 2px;
+        stroke-linecap: round;
+        fill: transparent;
+        transform-origin: 50%;
+        transform: rotate(-90deg);
+
+        &.has-stroke-warning {
+          stroke: $tertiary;
+        }
+      }
+    }
+  }
+
+  &-info {
+    flex-grow: 1;
+    padding-left: $gi-spacer;
+
+    &-title {
+      margin-bottom: $gi-spacer-xs;
+      line-height: 1;
+    }
+  }
+
+  @include tablet {
+    flex-wrap: nowrap;
+    justify-content: space-between;
+
+    &-body {
+      margin-bottom: 0;
+    }
+
+    &-info {
+      padding: 0 $gi-spacer 0 $gi-spacer-lg;
+    }
+  }
+}
+
+.gi-feedback {
+  margin-top: $gi-spacer;
+}
+
+.buttons {
+  flex-wrap: nowrap;
+  justify-content: flex-end;
+}
+</style>
+<script>
+import ButtonCountdown from './ButtonCountdown.vue'
+import Lang from '../utils/translations'
```

Comment:
This must be `L`, not `Lang`, because the strings are generated by searching for `L( .* )`

Reply:
fixed